kvbc/src/pruning_reserved_pages_client.cpp

```diff
@@ -23,7 +23,7 @@ static char* data(std::vector<std::uint8_t>& v) { return reinterpret_cast<char*>
 static const char* cdata(const std::vector<std::uint8_t>& v) { return reinterpret_cast<const char*>(v.data()); }
 
 static std::uint32_t toSecondsCount(const std::chrono::seconds& s) {
-  if (s.count() <= 0 || s.count() > std::numeric_limits<std::uint32_t>::max()) {
+  if (s.count() > std::numeric_limits<std::uint32_t>::max()) {
```

Comment:
Does it make sense to document that `batch_blocks_num = 0` and `s.count() <= 0` means pruning stop? Also, do we need to handle `s.count() == 0` somewhere else in a special way?

I think, in the future, it might make more sense to persist a different data structure (maybe an `oneof` or have an explicit flag that says pruning is stopped) instead of relying on these special values.

Reply:
@yontyon, what do u think? Is this better to have it as 0 or another flag?